src/blocks/buttons-child/edit.js

```diff
@@ -23,6 +23,7 @@ const ButtonsChildComponent = ( props ) => {
 		setAttributes,
 		name,
 		deviceType,
+		context
```

Comment:
Always add trailing comma.

```suggestion
		context,
```

Reply:
This coding practice has been implemented [in this PR](https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3283).

This conversation can be resolved once that PR is merged.